newclient/src/apollo/schemas/api.json

```diff
@@ -1661,7 +1661,7 @@
             "deprecationReason": null
           },
           {
-            "name": "price_change_percentage_24h",
+            "name": "price_change_24h",
```

Comment:
hey why was this changed?

Reply:
@olgakup hrmmm this must have been mixed up with some other file cause I don't think i touched this, do I just go ahead and change it directly on this file? getLatestPrices.ts still seems to have the correct attribute